ONSdigital/sdx-gateway
# Fix healthcheck

https://trello.com/c/FhwDYnNC/1009-investigate-sdx-gateway-healthcheck

Fixes healthcheck for the rabbit connections and the logging around it.  Previously the pika logging was at error only which reduced noise in the logs but made it so that it was possible to miss out on information.

Also renamed the healthcheck delay to `RABBITMQ_HEALTHCHECK_DELAY` and set it to a more sensible default value.  It's called this instead of `SDX_GATEWAY_HEALTHCHECK_DELAY` as other services might (and should, in the case of seft-consumer-service) want to use the value for their healthchecks if we do more in the future.

## How to test
- Ensure submissions can still flow through (via console)
- Check that sdx-gateway heartbeats in the same way as the other sdx services
- Ensure the rabbit connection task is running (should log lines that look like `2019-07-11T09:54:09.000051Z|INFO: sdx-gateway: 2019-07-11 09:54.09 Rabbit MQ health check response status=ok` every 30 or so seconds?) . There should be 3 of them as it's now checking 3 separate urls
